packages/console/src/components/CodeEditor/index.tsx

```diff
@@ -2,29 +2,31 @@ import MonacoEditor from '@monaco-editor/react';
 import * as monaco from 'monaco-editor';
 import React from 'react';
 
+import Copy from '@/icons/Copy';
+
+import IconButton from '../IconButton';
 import * as styles from './index.module.scss';
 
 type Props = {
   language: string;
-  isDarkMode?: boolean;
   height?: string;
   isReadonly?: boolean;
   value?: string;
   onChange?: (value: string) => void;
 };
 
-const CodeEditor = ({
-  value,
-  onChange,
-  language,
-  height = '300px',
-  isReadonly = false,
-  isDarkMode,
-}: Props) => {
+const CodeEditor = ({ value, onChange, language, height = '300px', isReadonly = false }: Props) => {
   const handleChange = (changedValue = '') => {
     onChange?.(changedValue);
   };
 
+  const handleCopy = async () => {
```

Comment:
Does it act like the `CopyToClipBoard`?

Reply:
NO, it doesn't have a toast and label, and different styles.

Reply:
toast/label will be better? we can leave this pr as it is and discuss with gumian offline